### DISCUSSION TITLE
Alien Swarms dates, lorewise

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#113] [Q&A] [answered]
I started to translate the new nominate for the official campaigns, until I realized,
this is the first time (as far as I know) somebody interpreted the date formats in the lore.

I always hesitated to translate something like this `Date: 20/06/32` (acc_info1_english.txt),
because I worked with enough date formats in my life to know there will be a minimum of one problem for sure:
I have no Idea in which millennium I am (letting alone interpretation of the used delimiter).

So? Where are we? Timewise.

**Comment by BenLubar** [comment]
Accident 32 takes place in 2032. It looks like most campaigns are either in the 2250s or the 2050s

**Reply by Duke-M-commits** [reply]
Wait...
so the marines must have some way to replicate for 200 years but still run around with the same tech?

**Reply by BenLubar** [reply]
I think it's just intentionally left ambiguous in most cases.

**Comment by Duke-M-commits** [comment, marked as answer]
Commit: 2812722d499aaafc49e88e5170d61af1a8a9db59
Reference: https://steamcommunity.com/app/563560/discussions/0/3421062958699939882/
Reference2: https://github.com/ReactiveDrop/reactivedrop_translations/wiki/Date-formats-&-time-ranges-used#the-right-century
